sdk/storage/azblob/CHANGELOG.md

```diff
@@ -1,14 +1,17 @@
 # Release History
 
-## 0.5.0 (Unreleased)
+## 0.5.0 (Beta)
```

Comment:
The part in the parens is the date of release.  So if we want to release this today it would be `(2022-09-26)`